packages/prosemirror-lwdita-backend/src/tests/github.controller.spec.ts

```diff
@@ -0,0 +1,190 @@
+/*!
+Copyright (C) 2020 Evolved Binary
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { Request, Response } from 'express';
+import { authenticateUserWithOctokit, commitChangesAndCreatePR, getUserInformation } from '../api/controller/github.controller';
+
+describe('authenticateUserWithOctokit', () => {
+  let req: Partial<Request>;
+  let res: Partial<Response>;
+  let jsonStub: sinon.SinonStub;
+  let statusStub: sinon.SinonStub;
+
+  beforeEach(async () => {
+    // Mocking req and res objects
```

Comment:
Please remove comments that don't add any value from the test.

Reply:
these comments: Arrange ,ACT, Assert are really connected to the stubbing methodology and I strongly recommend we should keep them in the tests.